lib/cli.js

```diff
@@ -9,8 +9,19 @@ const configuration = getConfig();
 const choicesList = choices(configuration);
 const questionsList = questions(choicesList);
 
-inquirer.prompt(questionsList).then((answers) => {
-  const message = answers.moreInfo ? `${answers.editor}` : `${answers.type} ${answers.description}`;
+const prompt = inquirer.prompt(questionsList);
+
+prompt.ui.process.subscribe((answer) => {
+  if (answer.name === 'commitMessageWarning') {
+    const continueExecution = answer.answer;
+    if (!continueExecution) {
+      process.exit();
```

Comment:
Maybe we should not `process.exit()` the entire cli (especially not without any exit code). I guess it would be better to put this validation into `promptConfig.js` -> `validate`. There we can warn the user to shorten their commit message, as we did `input => (input ? true : 'A commit message is mandatory!')`.

It is planned to add a new config option in `.sgcrc`:

```json
{
  "rules": {
    "end-with-dot": false
  }
}
```

With this option we can decide if we show the warning or not